sdk/cognitivelanguage/Azure.AI.Language.Conversations/README.md

```diff
@@ -469,32 +469,32 @@ foreach (var result in jobResults.Tasks.Items)
 First, you should prepare the input:
 
 ```C# Snippet:StartAnalyzeConversation_ConversationPII_Transcript_Input
-var transciprtConversationItemOne = new TranscriptConversationItem(
-   id: "1",
-   participantId: "speaker",
-   itn: "hi",
-   maskedItn: "hi",
-   text: "Hi",
-   lexical: "hi");
+var transciprtConversationItemOne = new TranscriptConversationItem(id: "1", participantId: "speaker")
```

Comment:
nit spelling in this sample for the variables

```suggestion
var transcriptConversationItemOne = new TranscriptConversationItem(id: "1", participantId: "speaker")
```

Reply:
No more models to misspell. :smile: